pages/docs/tutorials/kafka/configure-kafka-avro.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Create AsyncAPI Document for applications interacting with Kafka using Avro Schema
```

Comment:
first tutorial is about creating AsyncAPI document

this one is more about modifying it to use Avro

```suggestion
title: Describe Kafka message payload using Avro Schema
```

?